lib/hanami/slice_registrar.rb

```diff
@@ -85,6 +82,16 @@ def with_nested
 
     private
 
+    def ancestors
+      list = []
+      ancestor = parent
+      while ancestor
+        list.unshift(ancestor)
```

Comment:
Can we please use `prepend` here instead of `unshift`? I don't know about you, but `unshift` always breaks my brain. And for a very abstract method like this, which is already encroaching into brain-breaking territory, the kinder we can be to the readers' brains, the better 😄